src/components/dialog/pages/EnterPassword.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <dialog-content>
+    <template v-slot:header
+      >Unlock Switch</template
+    >
+    <p class="p">Enter your password to decrypt stored credentials and keys.</p>
```

Comment:
I would like a minimum password size.

Reply:
Minimum is now 8 characters. In the future, may want to integrate an algo to require passphrases with a certain amount of entropy.